xml/geo_booth_i.xml

```diff
@@ -64,6 +64,75 @@
    parties.
   </para>
  </sect1>
+
+ <sect1 xml:id="sec.ha.geo.booth.ticket-types">
+  <title>Automatic versus Manual Tickets</title>
+  <para>
+   A ticket grants the right to run certain resources on a specific
+   cluster site. Two types of tickets are supported:</para>
+  <itemizedlist>
+   <listitem>
+    <para>Automatic tickets are controlled by the &boothd; daemon.</para>
```

Comment:
Perhaps, we could add "and RAFT algorithm" or something similar?

Reply:
Hm, I'm hesitant to mention the algorithm. IMHO it's an implementation detail which does not matter to the administrator. In order to focus on the target group of the Geo Clustering Guide (admins), I would prefer not to mention it.

Reply:
Agreed with @taroth21